src/main/java/net/snowflake/ingest/streaming/internal/BlobBuilder.java

```diff
@@ -200,28 +210,24 @@ static byte[] buildBlob(
                 + BLOB_CHECKSUM_SIZE_IN_BYTES
```

Comment:
all the changes in this method (buildBlob) look unnecessary, is something wrong with the old logic?

Reply:
Merged by mistake, reverted.